packages/features/users/components/UserTable/UserListTable.tsx

```diff
@@ -269,18 +284,12 @@ export function UserListTable() {
         searchKey="member"
         selectionOptions={[
           {
-            label: "Add To Team",
-            onClick: () => {
-              console.log("Add To Team");
-            },
-            icon: Users,
+            type: "render",
+            render: (table) => <TeamListBulkAction table={table} />,
```

Comment:
Add the option to render components in the bar instead of just onclick handlers.

Easily allows us to use dialogTrigger to get the correct position of the popover to open. 

I tried without and ended up overcomplicating the calculation with refs :/

Reply:
![image](https://github.com/calcom/cal.com/assets/1780212/ecbb65df-d919-4052-9706-83c475db1e74)

Team that the users are already a member of are shown in Bulk Add to Team. This is the case even when all the selected users were part of that team. I think we should disable that team or not show at all.

But when some of them are a part of it and some of them are not, it currently says that all the users have been invited when infact only those that are not part of the team are added to the team. Maybe we can handle it by showing something like '2 members already a part and 3 were added'